cpp/src/arrow/csv/options.h

```diff
@@ -129,6 +129,10 @@ struct ARROW_EXPORT ConvertOptions {
 
   /// Create conversion options with default values, including conventional
   /// values for `null_values`, `true_values` and `false_values`
+  ///
+  /// Default null values: see http://crossbow.voltrondata.com/pr_docs/48048/cpp/csv.html#nulls
```

Comment:
We can not link to the preview docs. Here, in the header file, we can actually add the list of the conventional null values. Then this section can be linked from the csv.html.